html-css/week2/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WEB DEV</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="title_section-grey">
```

Comment:
For CSS class names stick to the kebab case instead of combining it with the snake case. It will make more sense when you start to use Sass with nested CSS. Then you'll experience the advantage of kebab case.